Specification/5._Application_level_conventions.md

```diff
@@ -56,7 +54,7 @@ In the case of cameras, the following convention should be preferred:
 
 For world frames, the North-East-Down (NED) notation should be preferred.
 
-The default rotation representation is quaternion, the coefficients are ordered as follows: X, Y, Z, W.
+The default rotation representation is quaternion. The coefficients are ordered as follows: X, Y, Z, W.
 
 Angular velocities should be represented in fixed axis roll (about X), pitch (about Y), and yaw (about Z).
```

Comment:
@pavel-kirienko What does "fixed axis roll" mean - specifically what does the "fixed" bit imply?

Reply:
It means that the axes of rotation are fixed while the object is rotated. See https://en.wikipedia.org/wiki/Rotation_around_a_fixed_axis. The sentence should read like "Angular velocities should be represented in fixed axis roll, pitch, and yaw". Do you think it should be rephrased?

Reply:
No. The current version is OK. Reading the linked article I pretty much assume all roll is around a fixed axis, so I just wasn't sure how this might differ from "normal".